src/test/java/L5task2/FizzBuzzGameTest.java

```diff
@@ -0,0 +1,27 @@
+package L5task2;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class FizzBuzzGameTest {
+
```

Comment:
Tests are wrong, you rewrote here your functionality to check it. Your fizzbuzz functions should be written in such a way that you should call it inside your tests